pkg/controller/node/nodecontroller.go

```diff
@@ -532,6 +532,32 @@ func (nc *NodeController) tryUpdateNodeStatus(node *api.Node) (time.Duration, ap
 				readyCondition.LastTransitionTime = nc.now()
 			}
 		}
+
+		// Like NodeReady condition, NodeOutOfDisk was last set longer ago than gracePeriod, so update
+		// it to Unknown (regardless of its current value) in the master, without contacting kubelet.
+		oodCondition := nc.getCondition(&node.Status, api.NodeOutOfDisk)
```

Comment:
Please add a TODO to refactor this with readyCondition to remove duplicated code